hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -90,12 +105,16 @@ public void migrate(@NonNull final MigrationContext ctx) {
         final var adminKey = getAccountAdminKey(ctx);
         final var nodeDetailMap = getNodeAddressMap(ctx);
 
-        Key finalAdminKey = adminKey == null || adminKey.equals(Key.DEFAULT)
+        final var defaultAdminKey = adminKey == null || adminKey.equals(Key.DEFAULT)
                 ? Key.newBuilder().ed25519(bootstrapConfig.genesisPublicKey()).build()
                 : adminKey;
         NodeAddress nodeDetail;
         final var addressBook = networkInfo.addressBook();
         for (final var nodeInfo : addressBook) {
+            final var nodeAdminKey = nodeAdminKeys.getOrDefault(nodeInfo.nodeId(), defaultAdminKey);
+            if (nodeAdminKey != defaultAdminKey) {
```

Comment:
Should this be `==`?
From how I read this, to log _when the override is used_ it would need to be `==`.
As it stands, it logs here when the admin key from the `nodeInfo` is used and the "default" is not applied.